walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/index.html

```diff
@@ -46,25 +47,58 @@ <h2>List of Projects I've worked on<h2>
             </p>
         </div>
     </div>
-    <div class="content-bg-3" id="content-bg">
-      	<section id="third-page" class="full-page">
-            <h1>Making change to see changes
-            </h1>
-            <p>
-                Nam at lectus urna duis convallis convallis tellus id. Tortor dignissim convallis 
-                aenean et tortor. Montes nascetur ridiculus mus mauris vitae ultricies leo. 
-                Praesent tristique magna sit amet purus gravida. Nulla facilisi cras fermentum 
-                odio eu feugiat. In hendrerit gravida rutrum quisque. Odio euismod lacinia at quis 
-                risus sed. Quis viverra nibh cras pulvinar mattis nunc sed blandit. Vitae tortor 
-                condimentum lacinia quis vel eros donec. Aliquam nulla facilisi cras fermentum 
-                odio eu feugiat pretium nibh. Odio eu feugiat pretium nibh ipsum. Aliquet bibendum 
-                enim facilisis gravida neque. Quam vulputate dignissim suspendisse in est ante in 
-                nibh mauris.
-            </p>
-        </section>
-    </div>
-      <p>Click <a href="images.html">here</a> to see more pictures of Stanley!</p>
-      <p>Click <a href="random.html">here</a> to see a random picture of Stanley!</p>
+
+
+<ul class="grid cs-style-1">
+    <li>
+        <figure>
+    	    <img src="images/profilepic.jpg" alt="img01">
+            <figcaption>
+				<h3>Image of Jalyn in front of Christmas Tree</h3>
```

Comment:
nit: html children elements should have either 2 spaces or 4 spaces indentation extra then their parent element. This makes it easier to follow the flow.

Reply:
Resolved!